projects/hslayers/src/components/compositions/dialogs/dialog_overwriteconfirm.html

```diff
@@ -2,26 +2,30 @@
     <div class="modal-dialog">
         <div class="modal-content">
             <div class="modal-header">
-                <h4 class="modal-title">{{'COMPOSITIONS.dialogoverwriteConfirm.overwriteExistingMap' | translateHs: {app: data.app} 
+                <h4 class="modal-title">{{'COMPOSITIONS.dialogoverwriteConfirm.overwriteExistingMap' | translateHs:
+                    {app: data.app}
                     }}</h4>
                 <button type="button" (click)="close()" class="btn-close" data-dismiss="modal"
-                    attr.aria-label="{{'COMMON.close' | translateHs: {app: data.app} }}">
+                    [attr.aria-label]="'COMMON.close' | translateHs: {app: data.app} ">
                 </button>
             </div>
             <div class="modal-body" style="max-height: 600px; overflow-y: auto">
                 {{'COMPOSITIONS.dialogoverwriteConfirm.youAreOpeningMap' | translateHs: {app: data.app} }}
                 <b>{{data.composition_name_to_be_loaded}}</b>
-                <br>{{'COMPOSITIONS.dialogoverwriteConfirm.thereAreCurrentlyUnsavedChanges' | translateHs: {app: data.app} }}
+                <br>{{'COMPOSITIONS.dialogoverwriteConfirm.thereAreCurrentlyUnsavedChanges' | translateHs: {app:
+                data.app} }}
                 <br>{{'COMPOSITIONS.dialogoverwriteConfirm.whatDoYouWant' | translateHs: {app: data.app} }}
             </div>
             <div class="modal-footer">
                 <button type="button" class="btn btn-primary compositions-btn-overwrite" (click)="overwrite()"
                     data-dismiss="modal">{{'COMMON.continue' | translateHs: {app: data.app} }}</button>
                 <button type="button" class="btn btn-primary compositions-btn-add" (click)="add();"
-                    data-dismiss="modal">{{'COMPOSITIONS.dialogoverwriteConfirm.mergeCompositions' | translateHs: {app: data.app} 
+                    data-dismiss="modal">{{'COMPOSITIONS.dialogoverwriteConfirm.mergeCompositions' | translateHs: {app:
+                    data.app}
```

Comment:
Prettier formatter?

Reply:
@jmacura Yes, a have set to format after save, just didn't expect so many occurrences.

Reply:
That's fine. I just dislike the way it wraps elements in HTML templates. But that's more my problem than yours 😄

Reply:
Sry, next files updates will come with ESLint 😄.

Reply:
Not sure if ESLint is capable of formatting HTML... I generally use VS Code's default HTML formatter or intend manually. That is a bad habit, of course...

Reply:
Yes, You are right! I will just set default formatter to VS Code's for HTML files.